README.md

```diff
@@ -611,6 +666,21 @@ jira::proxy:
   proxyName: 'jira.example.co.za'
   proxyPort: '443'
 jira::contextpath: '/jira'
+jira::tomcat_additional_connectors:
+  8181:
+    relaxedPathChars: '[]|'
+    relaxedQueryChars: '[]|{}^&#x5c;&#x60;&quot;&lt;&gt;'
```

Comment:
is this correct? The string looks a bit strange

Reply:
Looks strange, but see https://github.com/voxpupuli/puppet-jira/blob/master/templates/server.xml.erb#L54 on multiple lines, and https://confluence.atlassian.com/kb/how-to-create-an-unproxied-application-link-719095740.html and https://confluence.atlassian.com/jirakb/jira-server-startup-fails-with-tomcat-misconfigured-error-958453799.html for details on why this is necessary.